net/xinetd/files/xinetd.init

```diff
@@ -62,7 +62,7 @@ config_cb() {
 			local option="$1"
 			local value="$2"
 
-			[ -n "$value" ] && echo -e "\t$option = $value" >> $GENERATED_CONF_FILE
+			[ -n "$value" -a "$option" != "name" ] && echo -e "\t$option = $value" >> $GENERATED_CONF_FILE
```

Comment:
Please use `&&` instead of `-a`

Reply:
Whats the advance of `-a`?  `value` should be in most cases not empty